### DISCUSSION TITLE
For post headers, Hide NSFW label from post headers+

### DISCUSSION THREAD

**Discussion by BD9Max** [Docile-Alligator/Infinity-For-Reddit#988] [Ideas]
![Screenshot_20220324-123413~3](https://user-images.githubusercontent.com/82726385/159903976-f513de89-5e06-46d9-929e-42b2969517d5.jpg)

When u hide post types, flairs, awards and so - there still is NSFW label taking precious screen space. How to hide that?

Also for post headers, req: separate r/subreddit and u/username links apart in post header - so that it's easier to press them.

^^^^^^ these in cards views

REQ: for gallery view: enhance it? From gallery to post to media view it's seems much. Either gallery or media view needs such as vote buttons.

**Comment by Wladefant** [comment]
Add each request to its own post please
